test/parallel/test-os.js

```diff
@@ -101,3 +101,21 @@ switch (platform) {
 
 var EOL = os.EOL;
 assert.ok(EOL.length > 0);
+
+
+var home = os.homedir();
+
+console.log('homedir = ' + home);
+assert.ok(home.length > 0);
+
+if (common.isWindows && process.env.USERPROFILE) {
+  assert.equal(home, process.env.USERPROFILE);
+  delete process.env.USERPROFILE;
+  assert.equal(home, os.homedir());
+  process.env.USERPROFILE = home;
+} else if (!common.isWindows && process.env.HOME) {
+  assert.equal(home, process.env.HOME);
+  delete process.env.HOME;
+  assert.equal(home, os.homedir());
```

Comment:
I'm not sure if this is a good assumption.  I'd just check that it still returns _something_, possibly something that looks like a path.